### ISSUE TITLE
Sorting entries on batch export

### ISSUE THREAD

**Issue by henning-gerhardt** [goobi/goobi-production#417]
While exporting a couple of newspaper processes through batch export resulting anchor files (`_anchor.xml` and `_year.xml`) containg references in no order. Presentation this files through Goobi.Presentation display this entries in wild ordering.

On batch exporting some kind of ordering for example on process title should be done.

**Comment by henning-gerhardt** [comment]
Some examples:

- unsorted
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div ADMID="AMD" CONTENTIDS="http://digital.slub-dresden.de/id425390144" DMDID="DMDLOG_0000" ID="LOG_0000" LABEL="Artistisches Notizenblatt" ORDERLABEL="Artistisches Notizenblatt" TYPE="newspaper">
      <mets:div DMDID="DMDLOG_0001" ID="LOG_0001" LABEL="1822" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18220101/ArtiNo_425390144-18220101_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0002" ID="LOG_0002" LABEL="1823" ORDERLABEL="1823" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18230317/ArtiNo_425390144-18230317_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0003" ID="LOG_0003" LABEL="1832" ORDERLABEL="1832" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18321220/ArtiNo_425390144-18321220_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0004" ID="LOG_0004" LABEL="1833" ORDERLABEL="1833" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331019/ArtiNo_425390144-18331019_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0005" ID="LOG_0005" LABEL="1830" ORDERLABEL="1830" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18300630/ArtiNo_425390144-18300630_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0006" ID="LOG_0006" LABEL="1831" ORDERLABEL="1831" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18310326/ArtiNo_425390144-18310326_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0007" ID="LOG_0007" LABEL="1824" ORDERLABEL="1824" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-1824053102/ArtiNo_425390144-1824053102_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0008" ID="LOG_0008" LABEL="1825" ORDERLABEL="1825" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18250726/ArtiNo_425390144-18250726_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0009" ID="LOG_0009" LABEL="1826" ORDERLABEL="1826" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18260715/ArtiNo_425390144-18260715_year.xml"/>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

- sorted
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div ADMID="AMD" CONTENTIDS="http://digital.slub-dresden.de/id425390144" DMDID="DMDLOG_0000" ID="LOG_0000" LABEL="Artistisches Notizenblatt" ORDERLABEL="Artistisches Notizenblatt" TYPE="newspaper">
      <mets:div DMDID="DMDLOG_0001" ID="LOG_0001" LABEL="1822" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18220101/ArtiNo_425390144-18220101_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0002" ID="LOG_0002" LABEL="1823" ORDERLABEL="1823" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18230317/ArtiNo_425390144-18230317_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0007" ID="LOG_0007" LABEL="1824" ORDERLABEL="1824" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-1824053102/ArtiNo_425390144-1824053102_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0008" ID="LOG_0008" LABEL="1825" ORDERLABEL="1825" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18250726/ArtiNo_425390144-18250726_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0009" ID="LOG_0009" LABEL="1826" ORDERLABEL="1826" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18260715/ArtiNo_425390144-18260715_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0005" ID="LOG_0005" LABEL="1830" ORDERLABEL="1830" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18300630/ArtiNo_425390144-18300630_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0006" ID="LOG_0006" LABEL="1831" ORDERLABEL="1831" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18310326/ArtiNo_425390144-18310326_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0003" ID="LOG_0003" LABEL="1832" ORDERLABEL="1832" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18321220/ArtiNo_425390144-18321220_year.xml"/>
      </mets:div>
      <mets:div DMDID="DMDLOG_0004" ID="LOG_0004" LABEL="1833" ORDERLABEL="1833" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331019/ArtiNo_425390144-18331019_year.xml"/>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

**Comment by henning-gerhardt** [comment]
example of `_year.xml`

- unsorted:
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div CONTENTIDS="http://digital.slub-dresden.de/id(prozesstitle)" DMDID="DMDLOG_0002" ID="LOG_0002" LABEL="Artistisches Notizenblatt" ORDERLABEL="Artistisches Notizenblatt" TYPE="newspaper">
      <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18330330/ArtiNo_425390144-18330330_anchor.xml"/>
      <mets:div ADMID="AMD" DMDID="DMDLOG_0003" ID="LOG_0003" LABEL="1833" TYPE="year">
        <mets:div ADMID="AMD" DMDID="DMDLOG_0004" ID="LOG_0004" ORDERLABEL="3" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0005" ORDERLABEL="30" TYPE="day">
            <mets:div ID="LOG_0006" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18330330/ArtiNo_425390144-18330330.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0006" ID="LOG_0007" ORDERLABEL="11" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0008" ORDERLABEL="2" TYPE="day">
            <mets:div ID="LOG_0009" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331102/ArtiNo_425390144-18331102.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0010" ORDERLABEL="21" TYPE="day">
            <mets:div ID="LOG_0011" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331121/ArtiNo_425390144-18331121.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0009" ID="LOG_0012" ORDERLABEL="10" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0013" ORDERLABEL="19" TYPE="day">
            <mets:div ID="LOG_0014" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331019/ArtiNo_425390144-18331019.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0011" ID="LOG_0015" ORDERLABEL="12" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0016" ORDERLABEL="24" TYPE="day">
            <mets:div ID="LOG_0017" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331224/ArtiNo_425390144-18331224.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0018" ORDERLABEL="31" TYPE="day">
            <mets:div ID="LOG_0019" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331231/ArtiNo_425390144-18331231.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0020" ORDERLABEL="12" TYPE="day">
            <mets:div ID="LOG_0021" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331212/ArtiNo_425390144-18331212.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

- sorted
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div CONTENTIDS="http://digital.slub-dresden.de/id(prozesstitle)" DMDID="DMDLOG_0002" ID="LOG_0002" LABEL="Artistisches Notizenblatt" ORDERLABEL="Artistisches Notizenblatt" TYPE="newspaper">
      <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18330330/ArtiNo_425390144-18330330_anchor.xml"/>
      <mets:div ADMID="AMD" DMDID="DMDLOG_0003" ID="LOG_0003" LABEL="1833" TYPE="year">
        <mets:div ADMID="AMD" DMDID="DMDLOG_0004" ID="LOG_0004" ORDERLABEL="3" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0005" ORDERLABEL="30" TYPE="day">
            <mets:div ID="LOG_0006" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18330330/ArtiNo_425390144-18330330.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0009" ID="LOG_0012" ORDERLABEL="10" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0013" ORDERLABEL="19" TYPE="day">
            <mets:div ID="LOG_0014" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331019/ArtiNo_425390144-18331019.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0006" ID="LOG_0007" ORDERLABEL="11" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0008" ORDERLABEL="2" TYPE="day">
            <mets:div ID="LOG_0009" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331102/ArtiNo_425390144-18331102.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0010" ORDERLABEL="21" TYPE="day">
            <mets:div ID="LOG_0011" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331121/ArtiNo_425390144-18331121.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
        <mets:div ADMID="AMD" DMDID="DMDLOG_0011" ID="LOG_0015" ORDERLABEL="12" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0020" ORDERLABEL="12" TYPE="day">
            <mets:div ID="LOG_0021" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331212/ArtiNo_425390144-18331212.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0016" ORDERLABEL="24" TYPE="day">
            <mets:div ID="LOG_0017" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331224/ArtiNo_425390144-18331224.xml"/>
            </mets:div>
          </mets:div>
          <mets:div ADMID="AMD" ID="LOG_0018" ORDERLABEL="31" TYPE="day">
            <mets:div ID="LOG_0019" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/ArtiNo_425390144-18331231/ArtiNo_425390144-18331231.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

**Comment by henning-gerhardt** [comment]
Fixed with merged pull-request #423
